math-plugin/src/test/java/com/github/underscore/math/MathTest.java

```diff
@@ -287,6 +287,8 @@ public void chain() {
             public Boolean apply(String str) { return true; } });
         $.chain(new String[] {""}).rejectIndexed(new PredicateIndexed<String>() {
             public boolean apply(int index, String str) { return true; } });
+        $.chain(new String[] {""}).filterFalse(new Predicate<String>() {
```

Comment:
\[FindBugs\] INFO: RV: Return value of $$Chain\.filterFalse\(Predicate\) ignored, is this OK in com\.github\.underscore\.math\.MathTest\.chain\(\)